src/Fixer.php

```diff
@@ -149,6 +150,8 @@ public function fixFile()
 
         $this->enabled = true;
 
+        StatusWriter::pause();
```

Comment:
I suggest adding a comment explaining why it is necessary to call `StatusWritter::pause()` here, similar to what you did in `AbstractPatternSniff::createTokenPattern()`.